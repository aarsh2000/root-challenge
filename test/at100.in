Driver McQueen
Trip McQueen 1:00 2:00 100.00