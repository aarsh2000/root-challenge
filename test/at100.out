McQueen: 100 miles @ 100 mph
